stytch/b2b/discovery/organizations/types.go

```diff
@@ -0,0 +1,197 @@
+//     debug an issue.
+//   - MemberID: Globally unique UUID that identifies a specific Member.
+//   - SessionToken: A secret token for a given Stytch Session.
+//   - SessionJWT: The JSON Web Token (JWT) for a given Stytch Session.
+//   - Member: The [Member object](https://stytch.com/docs/b2b/api/member-object).
+//   - StatusCode: The HTTP status code of the response. Stytch follows standard HTTP response status code
+//     patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX
+//     are server errors.
+//   - MemberSession: The [Session object](https://stytch.com/docs/b2b/api/session-object).
+//   - Organization: The [Organization object](https://stytch.com/docs/b2b/api/organization-object).
+type CreateResponse struct {
+	RequestID     string                     `json:"request_id,omitempty"`
+	MemberID      string                     `json:"member_id,omitempty"`
+	SessionToken  string                     `json:"session_token,omitempty"`
+	SessionJWT    string                     `json:"session_jwt,omitempty"`
+	Member        organizations.Member       `json:"member,omitempty"`
+	StatusCode    int32                      `json:"status_code,omitempty"`
+	MemberSession sessions.MemberSession     `json:"member_session,omitempty"`
+	Organization  organizations.Organization `json:"organization,omitempty"`
+}
+
+// ListResponse: Response type for `Organizations.List`.
+// Fields:
+//
+//   - RequestID: Globally unique UUID that is returned with every API call. This value is important to log
+//     for debugging purposes; we may ask for this value to help identify a specific API call when helping you
+//     debug an issue.
+//
+//   - EmailAddress: The email address.
+//
+//   - DiscoveredOrganizations: An array of `discovered_organization` objects tied to the
+//     `intermediate_session_token`, `session_token`, or `session_jwt`. See the [Discovered Organization
+//     Object](https://stytch.com/docs/b2b/api/discovered-organization-object) for complete details.
+//
+//     Note that Organizations will only appear here under any of the following conditions:
+//     1. The end user is already a Member of the Organization.
+//     2. The end user is invited to the Organization.
+//     3. The end user can join the Organization because:
+//
+//     a) The Organization allows JIT provisioning.
+//
+//     b) The Organizations' allowed domains list contains the Member's email domain.
+//
+//     c) The Organization has at least one other Member with a verified email address with the same
+//     domain as the end user (to prevent phishing attacks).
+//
+//   - StatusCode: The HTTP status code of the response. Stytch follows standard HTTP response status code
+//     patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX
+//     are server errors.
+type ListResponse struct {
```

Comment:
nit: it's kind of annoying how in this file we don't go Request, Response. We go Request, Request, Response, Response

Reply:
I can change the sorting. Right now I specifically sort it as HelperType > Request > Response, but I can change that

Reply:
I can change the sorting. Right now I specifically sort it as HelperType > Request > Response, but I can change that